pkg/controller/account/account_controller.go

```diff
@@ -52,6 +53,20 @@ const (
 	AccountReady = "Ready"
 )
 
+// Custom errors
+
+// ErrAwsAccountLimit indicates the orgnization account limit has been reached.
+var ErrAwsAccountLimit = errors.New("ACCOUNT_LIMIT_EXCEEDED")
+
+// ErrAwsInternalFailure indicates that there was an internal failure on the aws api
+var ErrAwsInternalFailure = errors.New("INTERNAL_FAILURE")
+
+// ErrAwsFailedCreateAccount indicates that an account creation failed
+var ErrAwsFailedCreateAccount = errors.New("Failed to create account")
+
+// ErrAwsTooManyRequests indicates that to many requests were sent in a short period
+var ErrAwsTooManyRequests = errors.New("TooManyRequestsException")
```

Comment:
Why the 3 schemes for the arg to errors.New?  Can we be consistent and pick one or is there some functional reason?

Reply:
var ErrAwsAccountLimit = errors.New("ACCOUNT_LIMIT_EXCEEDED")
var ErrAwsInternalFailure = errors.New("INTERNAL_FAILURE")
var ErrAwsTooManyRequests = errors.New("TooManyRequestsException")

The above I took the names from aws , but I will standardize

The below was in the code before. Once I make sure no one is checking these error strings I will make this change as well.